web/src/use/use-user.ts

```diff
@@ -1,9 +1,9 @@
 import { type Ref, reactive, unref, watch, toRefs } from "vue"
 import { isNil } from "lodash"
 
-import usersApi, { type User, type UserUpdate } from "@/api/users-api"
+import usersApi, { RoleTypes, type User, type UserUpdate } from "@/api/users-api"
 
-export { type User, type UserUpdate }
+export { RoleTypes, type User, type UserUpdate }
```

Comment:
Consider adding `type RoleTypes` as this will play better with the Typescript + Vitest. (vitest is what we will likely be using for testing, once we do more of that)